presidio-analyzer/presidio_analyzer/conf/default.yaml

```diff
@@ -33,3 +33,4 @@ ner_model_configuration:
   - PRODUCT
   - QUANTITY
   - WORK_OF_ART
+  - SOME_LABEL
```

Comment:
Just making sure you wanted to add this label and keep the test

Reply:
Another good catch!